src/components/List/ListSplits.tsx

```diff
@@ -32,8 +32,9 @@ export function ListSplits({
             <span className={cs(style.pct)}>
               {displayPercentage(item.pct/1000)}%
             </span>
-            <div className={cs(style.user)}>
-              <UserBadge
+            <div>
+	      <UserBadge
+		className={style.user}
```

Comment:
Since the anchor remains the outermost container in the `<UserBadge />` we can simply overwrite styles by applying the class to the component directly